pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -77,7 +79,25 @@ This procedure describes how to create a key/value pair secret called `my-secret
    dcos security secrets create --value=top-secret developer/my-secret
    ```
 
-# Creating secrets from a file via the DC/OS Enterprise CLI
+# Creating secrets from a any file via the DC/OS Enterprise CLI
+
+This procedure describes how to use any file to create a secret called `myapp.keystore` inside the `developer` path using the DC/OS Enterprise CLI.
+
+The contents of the file (referred to below as `myapp.keystore`) can be any value, including binary and text value. The secrets service encodes binary files into `BASE64` text form before storing it in secret store.
+
+1. Use `dcos auth login` to log into the CLI.
+
+1. Use the following command to create the new secret.
+
+  ```bash
+  dcos security secrets create -f myapp.keystore developer/keystore
+  ```
+
+   **Important:** The maximum file size for a `BASE64` encoded file is approximately one MiB, subtracting approximately one KB for the secret store metadata.
+
+   **Important:** In previous versions of the `dcos-enterprise-cli` (bellow version `1.4.2`) the `-f` or `--file` flag was used to upload UTF-8 encoded text files as a text secrets. This has changed and using `-f` flag now treats the file as a general file stored on the filesystem and it is sent to the DC/OS Secrets service as a binary secret.
```

Comment:
s/bellow/below

Reply:
"a general file" - can this be made more specific?

Reply:
> "a general file" - can this be made more specific?
In what would you like to specify this?

Reply:
I don't know what it should be, but I'm fairly sure it should not be "a general file".

Maybe "using the `-f` flag will no longer make the `dcos-enterprise-cli` decode the given file as UTF-8 encoded text"